kubectl-rook-ceph.sh

```diff
@@ -566,6 +580,16 @@ function run_main_command() {
   esac
 }
 
+# Default values
+: "${ROOK_CLUSTER_NAMESPACE:=rook-ceph}"
+: "${ALTERNATE_IMAGE=}"
```

Comment:
This isn't a global value. IMO, this should be moved to the section where debug commands are parsed.

Reply:
I agree `ALTERNATE_IMAGE` this var is not a global variable.

Reply:
@subhamkrai can you look into this https://github.com/rook/kubectl-rook-ceph/pull/49#discussion_r966895863 as a reason to make it GLobal